CONTRIBUTING.md

```diff
@@ -10,6 +10,18 @@ We pledge to act and interact in ways that contribute to an open, welcoming, div
 
 You agree to follow our Code of Conduct while working on the project and/or being a part of any related communities. You can read that [here](https://github.com/codethesaurus/codethesaur.us/blob/main/CODE_OF_CONDUCT.md).
 
+## Testing
```

Comment:
> @geekygirlsarah Added some testing docs as well for the contribution page. All test cases are passing now.

I started having the README point to parts of the documentation for things like this so the README remains simpler and shorter and they can look up whatever they need. Would you mind adding that to do the docs repo instead to go with the issue there, and point the README to that? (Or just undo it and I'll try to add it tomorrow or Monday)

Reply:
I'll just remove the readme changes for now.